arbnode/schema.go

```diff
@@ -4,7 +4,6 @@
 package arbnode
 
 var (
-	BlockValidatorPrefix         string = "v"         // the prefix for all block validator keys
```

Comment:
util is very generic name and explicitly discouraged by go styleguide [0]. Such package is already created in this pr: storage, and also has todo to move this whole schema.go in there.

[0] https://google.github.io/styleguide/go/decisions#package-names

Reply:
storage is a fine name, it just doesn't belong in dataposter. We could create a storage package in the util folder. The goal of the util folder is just to avoid cluttering the top-level repo with miscellaneous independent libraries.